docker-compose.yaml

```diff
@@ -2,7 +2,7 @@ version: "3.2"
 
 services:
   starknet:
-    image: ghcr.io/keep-starknet-strange/madara:v0.1.0-testnet-sharingan-beta.7.1.experimental.4
+    image: ghcr.io/keep-starknet-strange/madara:v0.1.0-beta
```

Comment:
Depends on this job https://github.com/keep-starknet-strange/madara/actions/runs/5677451093/job/15385784124